models/lwr/17x17_uo2_assembly/rodded/make_openmc_model.py

```diff
@@ -0,0 +1,57 @@
+#--------------------------------------------------------------------------------------------------------------------------#
+# This is based on the C5G7 reactor physics benchmark problem extension case as described in:                              #
+# "Benchmark on Deterministic Transport Calculations Without Spatial Homogenisation: MOX Fuel Assembly 3-D Extension Case" #
+# [NEA/NSC/DOC(2005)16]                                                                                                    #
+# https://www.oecd-nea.org/upload/docs/application/pdf/2019-12/nsc-doc2005-16.pdf                                          #
+#                                                                                                                          #
+# The original C5G7 benchmark is defined with multi-group cross sections. To account for                                   #
+# continuous energy spectral effects, we chose to use the material properties provided in:                                 #
+# "Proposal for a Second Stage of the Benchmark on Power Distributions Within Assemblies"                                  #
+# [NEA/NSC/DOC(96)2]                                                                                                       #
+# https://www.oecd-nea.org/upload/docs/application/pdf/2020-01/nsc-doc96-02-rev2.pdf                                       #
+#--------------------------------------------------------------------------------------------------------------------------#
+
+import sys
+sys.path.append("../../")
+
+import numpy as np
+from argparse import ArgumentParser
+
+import openmc
+import openmc_common as geom
+from openmc_materials import MATERIALS as mats
+from openmc_assemblies import ASSEMBLIES as asmb
+from openmc_assemblies import assembly_bb, pins_per_axis
+
+ap = ArgumentParser()
+ap.add_argument('-n', dest='n_axial', type=int, default=1,
```

Comment:
Can we add something to the README.md that describes this flag? We're not really exercising it now, but it may be useful to have an explicit note about it.